README.md

```diff
@@ -101,8 +100,8 @@ Full list of options in `config.json`:
 | user                                | String  | Yes        | Snowflake User                                                |
 | password                            | String  | Yes        | Snowflake Password                                            |
 | warehouse                           | String  | Yes        | Snowflake virtual warehouse name                              |
-| aws_access_key_id                   | String  | Yes        | S3 Access Key Id                                              |
-| aws_secret_access_key               | String  | Yes        | S3 Secret Access Key                                          |
+| aws_access_key_id                   | String  | No         | S3 Access Key Id                                              |
```

Comment:
can you please add a small comment here, stg like:

Used for S3 to Snowflake copy operations. If not provided, credentials will be collected from the AWS_ACCESS_KEY_ID environment variable